app/commands/seed_local_command.rb

```diff
@@ -143,6 +156,18 @@ def mapping_rules_from(data)
         input_type, output_type = output_type, input_type
       end
       rules = raw_mappings["rules"]
+      input_taxonomy = if raw_mappings["input_taxonomy"] == "shopify"
+        shopify_taxonomy_version
+      else
+        raw_mappings["input_taxonomy"]
+      end
+
+      output_taxonomy = if raw_mappings["output_taxonomy"] == "shopify"
+        shopify_taxonomy_version
+      else
+        raw_mappings["output_taxonomy"]
+      end
```

Comment:
Now that we're using the unstable version in the mapping files, I don't think we need this block anymore since the input/output taxonomy will be `shopify/{version}-unstable`